ansible/library/topo_facts.py

```diff
@@ -206,7 +206,8 @@ def parse_topo_defintion(self, topo_definition, po_map, dut_num, neigh_type='VMs
                                 vmconfig[vm]['ipv6mask'][dut_index] = ip_mask if ip_mask else '128'
         return vmconfig
 
-    def get_topo_config(self, topo_name, hwsku, asic_name, asics_present, card_type):
+
+    def get_topo_config(self, topo_name, hwsku, testbed_name, asics_present, card_type):
```

Comment:
ansible/library/topo_facts.py:210:5: E303 too many blank lines (2) - precommit flake8 gave this error

Reply:
updated.